apps/xomg_tasks/mix.exs

```diff
@@ -0,0 +1,26 @@
+defmodule OMG.XomgTasks.MixProject do
+  @moduledoc """
+  This is just a proxy app to hold and use all the code related to running `xomg` Mix.Tasks.
+
+  NOTE: this is not a proper mix app, just some Mix.Tasks which calls into other mix apps
+  """
+  use Mix.Project
+
+  def project do
+    [
+      app: :xomg_tasks,
+      version: OMG.Umbrella.MixProject.umbrella_version(),
+      build_path: "../../_build",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
```

Comment:
variables `build_path`, `deps_path`, `lockfile`, `elixir_paths` are unnecessary

Reply:
Why?

They are necessary, for mix to be able to understand this app when ran individually (if we want to someday). Otherwise, mix builds in the wrong dir.

Also it's a standard and we do that in other `mix.exs`.

Reply:
your run this app from header project folder, 
If someday we want running individually this be weird to sed build_path so hight